README.md

```diff
@@ -1,6 +1,6 @@
 # Mastodon compose client
 [![CC-BY license](https://img.shields.io/badge/License-CC--BY-blue.svg)](https://creativecommons.org/licenses/by-nd/4.0)
-![CI](https://github.com/thebino/MastodonCompose/workflows/CI/badge.svg)
+![CI](https://github.com/AndroidDev-social/MastodonCompose/actions/workflows/continuous-delivery-pipeline.yml/badge.svg)
```

Comment:
This will break again once the following PR gets merged, which changes the name of the script
https://github.com/AndroidDev-social/MastodonCompose/pull/41

Reply:
Since it is not merged yet and will not soon without changes, we can approve this one.
Taking small steps is always better, then none

Reply:
Okay, then let's just make sure this is updated on that other PR that is open or done as a follow-up right away. I prefer to avoid churn on small stuff like this and just get it fixed and over with.

Reply:
approve and merge it, I'll have a look into the scripts